### DISCUSSION TITLE
tteck, please look here ~

### DISCUSSION THREAD

**Discussion by leo852** [tteck/Proxmox#2422] [General]
Hi tteck, don't got your mail, so wanna find you here and wanna ask that are you interested in putting some 3rd party scripts on your site? (Such as mine)

It's a script for pulling official linux cloud images and pack them into templates, currently debian 10/11/12, ubuntu 20/22, almalinux 8/9, CentOS 7/8s/9s are supported. There's also options for installing qemu-guest-images and enabling ssh / ssh password login.

**Comment by tteck** [comment]
Are you talking about the newly created _HOKOHOST/Proxmox-Cloudinit-Template-Downloader_ github project?

**Comment by tteck** [comment]
You are welcome to create a pull request for this repository. While there's nothing inherently wrong with your coding style, it does not align with the established patterns and conventions of this project. Therefore, a complete rewrite may be necessary to ensure compatibility with this project's standards.

**Comment by tteck** [comment]
Feel free to check out the script at `https://github.com/tteck/Proxmox/raw/main/vm/ubuntu-vm.sh`; it downloads and installs the same Ubuntu 22.04 image as your script. With a simple modification, my script could handle all OS images using a dialog box format. Realistically, including all those OS options might not be useful.
